### PR TITLE
Make testrunner succeed again on OS X with safari and chrome

### LINKED ISSUE #9027: 2 testrunner tests fail on OS X 10.11 with safari 9.1.1 and current master
Current master framework test with safari 9.1.1 shows 2 failed tests:

```1.``` Test fails in
https://github.com/qooxdoo/qooxdoo/blob/master/framework/source/class/qx/test/bom/Attribute.js#L169

```
qx.test.bom.Attribute:testRemoveAttribute
Error message is: 
: Expected '-1' but found '524288'!
Stack trace: 
qx.core.AssertionError.constructor:51
qx.core.Assert.__fail:45
qx.core.Assert.assertEquals:39
qx.type.BaseString.assertEquals:34
anonymous()
qx.dev.unit.TestResult.testRemoveAttribute:40
anonymous()
qx.dev.unit.TestResult.run:27
qx.dev.unit.TestFunction.prototype.run()
qx.dev.unit.TestLoaderBasic.prototype.runTests()
anonymous()
```

```2.``` Test fails in https://github.com/qooxdoo/qooxdoo/blob/master/framework/source/class/qx/test/io/request/XhrWithRemote.js#L137

```
qx.test.io.request.XhrWithRemote:test: progress phases when abort after loading
Error message is: 
Expected [opened, sent, loading, abort], but found [opened, sent, abort]: error
Stack trace: 
qx.core.AssertionError.constructor:51
qx.core.Assert.__fail:45
qx.core.Assert.fail:18
qx.core.Assert.assertArrayEquals:18
qx.type.BaseString.assertArrayEquals:39
qx.dev.unit.TestResult:40
qx.dev.unit.TestCase:38
qx.event.dispatch.Direct.run:38
qx.Interface.resume:36
qx.event.Manager:33
qx.event.Registration.dispatchEvent:31
.wrappedFunction:56
qx.io.request.AbstractRequest.dispatchEvent:25
qx.io.request.AbstractRequest.fireNonBubblingEvent:21
qx.io.request.AbstractRequest.fireDataEvent:30
qx.bom.request.Xhr._setPhase:27
qx.bom.request.Xhr._fireStatefulEvent:21
qx.bom.request.Xhr._onAbort:36
qx.bom.request.Xhr:30
qx.bom.request.Xhr._emit:29
qx.io.request.AbstractRequest.__readyStateChangeDone:29
qx.event.dispatch.Direct.__readyStateChange:38
qx.Interface.__onNativeReadyStateChange:36
qx.event.Manager:33
qx.event.Registration:51
.abort:45
qx.event.GlobalError:32
qx.event.Timer:28
```

### PR DESCRIPTION
**Pull request by level420**
Fixes #9027

### CODE REVIEW

**Comment by level420** [comment]
Pleas hold on: This is now failing on chrome with the qx.bom.Attribute tests.
Reworking the PR

**Comment by level420** [comment]
The testrunner now completes without failed tests on OS X 10.11.5 with safari 9.1.1, chrome 51, firefox 46.0.1

File: framework/source/class/qx/test/bom/Attribute.js [hunk 1]
```diff
@@ -165,6 +166,9 @@ qx.Class.define("qx.test.bom.Attribute",
       if (qx.core.Environment.get("browser.name") == "edge") {
         maxLengthValue = this.__maxLengthValues.mshtml;
       }
+      else if (qx.core.Environment.get("browser.name") == "chrome") {
+        maxLengthValue = this.__maxLengthValues["default"];
+      }
```
Comment (oetiker) [inline framework/source/class/qx/test/bom/Attribute.js]: not that I had any clue what was going on here

Comment (oetiker) [inline framework/source/class/qx/test/bom/Attribute.js]: ok so this is because chrome sais webkit while it should say blink ...

**Pull request merged**